build/yaml/botbuilder-dotnet-ci-slack-test.yml

```diff
@@ -67,13 +67,26 @@ steps:
   displayName: 'Set Environment Variables'
 
 - task: AzureCLI@1
-  displayName: 'Create resources, deploy bot'
+  displayName: 'Create resources'
+  inputs:
+    azureSubscription: $(AzureSubscription)
+    scriptLocation: inlineScript
+    inlineScript: |
+     :: Comment out either Option 1 or Option 2 below. Ultimately, both templates should be exercised to ensure they continue to work.
+     
+     :: Option 1: Use the "new-rg" template:
+     call az deployment sub create --name "$(BotGroup)" --template-file "$(System.DefaultWorkingDirectory)\tests\Adapters\Microsoft.Bot.Builder.Adapters.Slack.TestBot\DeploymentTemplates\template-with-new-rg.json" --location "westus" --parameters appId=$(AppId) appSecret="$(AppSecret)" botId="$(BotName)" botSku=F0 newAppServicePlanName="$(BotName)" newWebAppName="$(BotName)" groupName="$(BotGroup)" groupLocation="westus" newAppServicePlanLocation="westus" slackVerificationToken="$(SlackVerificationToken)" slackBotToken="$(SlackBotToken)"  slackClientSigningSecret="$(SlackClientSigningSecret)"
```

Comment:
This option will fail with a template validation error if the template is set to "outer scope".

Reply:
This template with "outer scope" is succeeding. No error. It just required that the "dependsOn" statements be properly formed.